src/nba/background.js

```diff
@@ -1,16 +1,9 @@
-chrome.runtime.onInstalled.addListener(function() {
-    chrome.declarativeContent.onPageChanged.removeRules(undefined, function() {
-        chrome.declarativeContent.onPageChanged.addRules([
-            {
-                conditions: [
-                    new chrome.declarativeContent.PageStateMatcher({
-                        pageUrl: { hostEquals: 'fantasy.espn.com', pathPrefix: '/basketball/team' },
-                    })
-                ],
-                actions: [ new chrome.declarativeContent.ShowPageAction() ]
-            }
-        ]);
-    });
+chrome.tabs.onUpdated.addListener(function(tabId, changeInfo, tab) {
```

Comment:
I think this part should be changed as follows:

    chrome.tabs.onUpdated.addListener(function (tabId, changeInfo, tab) {
        if (changeInfo.status === 'complete') {
            if (tab.url && tab.url.match(/fantasy.espn.com\/basketball\/team/)) {
                chrome.pageAction.show(tabId);
            } else {
                chrome.pageAction.hide(tabId);
            }
        }
    });

And of course similarly in the NHL code.

With the current code, the extension popup normally doesn't show up for me. If I understand the logic of this correctly, we need to show the page action when the URL matches and hide it when it doesn't, but only when the status is "complete". Sometimes you get other messages where the status is "progress" or simply undefined.

Reply:
What is the "extension popup"? I've only seen the buttons when I have used the extension so I might have missed that part -- what should I see, and how is it triggered?

Reply:
The popup you get when you click the extension icon while on the clubhouse page:
![image](https://user-images.githubusercontent.com/284189/115087270-24c23d00-9f0e-11eb-88f5-1f16a8c11b81.png)

With the current code it looks like this:
![image](https://user-images.githubusercontent.com/284189/115087384-589d6280-9f0e-11eb-88c5-6b7a1bfbc356.png)

Reply:
Hey @cameronkeif. Is my explanation clear? I expect that most people never use the popup page, but it does need to be there, at least for the "Set up for N days" feature.

Reply:
Yes it is clear - sorry, I just haven't been able to work on this lately.

Reply:
Ah, no worries. There's obviously no rush here. The NBA and NHL fantasy seasons are drawing to a close at this point anyway, so a possible Firefox version of the extension likely won't see any installs until the start of the next season.